### DISCUSSION TITLE
Help with being keyrolled

### DISCUSSION THREAD

**Discussion by 88411** [3kh0/ext-remover#1412] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I used to run shims all the time in the past without any issues. Now that down work. And yes I make sure to go into dev mode first. I have entered secure mode with wp disabled if that could be a reason. 

Any help is super appreciated

**Comment by 88411** [comment]
I talked with someone on discord and learned that there isn't much that I can really do about it 😔

**Reply by schoolexploitkid** [reply]
uh unenroll maybe?
